cloud/openstack/os_server_facts.py

```diff
@@ -64,10 +77,16 @@ def main():
 
     try:
         cloud = shade.openstack_cloud(**module.params)
-        server = cloud.get_server(module.params['server'])
-        hostvars = dict(openstack=meta.get_hostvars_from_server(
-            cloud, server))
```

Comment:
Is this part lost, `get_hostvars_from_server` ? Is that automatically grabbed by `list_servers` ?